chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -73,6 +73,25 @@
 
   include_recipe "crowbar::prepare-upgrade-scripts"
 
+when "none"
+
+  Chef::Log.debug("Deleting the scripts created for the upgrade...")
+  [
+    "prepare-repositories",
+    "upgrade-os",
+    "shutdown-services-before-upgrade",
+    "delete-cinder-services-before-upgrade",
+    "evacuate-host",
+    "pre-upgrade",
+    "delete-pacemaker-resources",
+    "router-migration",
+    "post-upgrade",
+    "chef-upgraded"
+  ].each do |f|
+    file "/usr/sbin/crowbar-#{f}.sh" do
+      action :delete
+    end
+  end
```

Comment:
Or is it better to do it from rails app once, so chef-client won't by trying to delete those files on each run?